### DISCUSSION TITLE
How can I drop the db using `prisma migrate reset` without losing Supabase permissions and configuration?

### DISCUSSION THREAD

**Discussion by lumenwrites** [supabase/supabase#10564] [Questions]
Hi! I'm struggling with an issue similar to the one described in this question:
https://stackoverflow.com/questions/67551593/supabase-client-permission-denied-for-schema-public

Sometimes I need to be able to drop the database using `prisma migrate reset`, but that breaks the permissions in my Supabase project. The answer in the SO thread above didn't work for me, besides, it is way too error prone and complicated to do every time.

Recreating Supabase project from scratch wouldn't work for me either - it takes me like 1-2 hours to set everythin up (the RLS policies, auth config, trigger functions, hooks, new API keys, etc, etc), and it's very time consuming and error prone.

Is there a way for me to drop the db and wipe the tables, but keep everything else about the supabase project intact?

**Comment by ar-corinaldi** [comment]
You should set up manually in the remote repo the RLS and all the config needed. Then, create and link the migration to your project using `supabase db diff --linked --use-migra -f <filename>` you get a new file with all the info. Use it to setup everything.
